contrib/bash/goshy

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -x
+show_help() {
+  cat <<EOF
+Usage: goshy [Options] FILE
+Options:
+  -b --burn              Burn after reading.
+  -h --help              Print this help message.
+  -u --only-url          Print only URL as response.
+  -p --period TIMEFRAME  Set a custom expiry period.
+EOF
+}
+
+while getopts p:period: flag
+do
+  case "$\{flag\}" in
+    p) PERIOD_IS_SET=1;;
+    period) PERIOD_IS_SET=1;;
+  esac
+done > /dev/null 2>&1
+
+if [ -z $PERIOD_IS_SET ]; then
+  PERIOD='3d'
+fi
+
+while :; do
+  case "$1" in
+    -p|--period)
+      PERIOD=$2
+      shift
+      ;;
+
+    -b|--burn)
+      BURN='-F burn=1'
+      ;;
+
+    -u|--only-url)
+      ONLYURL='?onlyURL'
+      ;;
+
+    -h|--help)
+      show_help
+      exit 0
+      ;;
+
+    *)
+      FILE=$1
+      break
+  esac
+
+  shift
+done
+
+PERIOD="-F period=$PERIOD"
+
+if [[ -z "$1" ]]; then
+  show_help
+  exit 1
+fi
+
+curl $GOSH_INSTANCE/$ONLYURL -F "file=@$1" $PERIOD $BURN 
```

Comment:
1. The final file should be `$FILE` nor `$1`, right?
2. Please double quote all variables.

```suggestion
curl "${GOSH_INSTANCE}/${ONLYURL}" -F "file=@${FILE}" "$PERIOD" "$BURN"
```

Reply:
Good catch, resolved with 8cb43602bbe9cd1300b9855adab62a342319b308.